### PR TITLE
South dox patch 1

### PR DESCRIPTION
**Pull request by SouthDox**

### CODE REVIEW

File: src/plugins/Typso.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import { addPreSendListener } from "@api/MessageEvents";
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+
+const keys = ["qwertyuiop", "asdfghjkl", "zxcvbnm"];
+
+export default definePlugin({
+    name: "Typso",
+    description: "Mke yoyr fruends thibk you'te typinf woth yoyr toes!",
+    version: "1.0.0",
```
Comment (somerandomcloud) [inline src/plugins/Typso.ts:28]: Version doesn't exist in plugin definiton

File: src/plugins/Typso.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    authors: [Devs.mantikafasi],
+    start() {
+
+        addPreSendListener((_, msg) => {
+            var newMsg = "";
+            let guh = false;
```
Comment (somerandomcloud) [inline src/plugins/Typso.ts:34]: Use something more descriptive than `guh`, it took me a whole minute to decipher what guh was meant to do, and still dont fully understand. Use something like `isLink`

**Review by somerandomcloud — COMMENTED** [review]
Also, your pr should have at least the plugin name/what you are adding, like so: `feat(Plugin): Typso`

You should also add a short description if possible. Other than my small suggestions, everything looks relatively fine, but I am sure there are some small improvements available but cant see any rn. Thanks for the pr!

**Comment by Vendicated** [comment]
what

**Comment by somerandomcloud** [comment]
didnt see this was a literal copypaste lmfao

**Pull request closed without merging**